services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user_model = get_user_model()
+    user = user_model.objects.get(username=username)
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket_data in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket_data["movie_session"],
+                order=order,
+                row=ticket_data["row"],
+                seat=ticket_data["seat"]
+            )
+    return order
```

Comment:
The `create_order` function correctly uses a transaction to ensure that the creation of an order and its associated tickets is atomic. This helps maintain data integrity.